wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -129,6 +144,12 @@ <h1>{{ stations[0].station_nm }}</h1>
                 </div>
                 {% if request.user_agent.browser == 'msie' %}
                     <div id="static-graph-div"></div>
+                    <div class="provisional-data-statement">
+                        <p>
+                            <span class="usa-tag">Important</span>
+                            data may be provisional - <a class="usa-link" href="{{ url_for('provisional_data_statement') }}">learn more</a>
```

Comment:
Can we capitalize Data?

Reply:
Sorry, yeah, I lean toward skipping capitalization when possible (to save space (and links look better too)), but you are right, the rest of page uses capitals so I'll change.